### ISSUE TITLE
Sidebar and legend lack accessibility labels

### ISSUE THREAD

**Issue by etong11** [rewiringamerica/incentives-visualizer#93]
Tasks:
- Add accessibility labeling to sidebar
- Add accessibility labeling to legend
- Fix legend to overlay outside of map container rather than inside (not intractable currently)
- Fix sidebar filtering options to be navigable using keyboard controls

Verification:
- Labels can be read by a screen reader for the legend and sidebar
- Sidebar filtering options can be selected/deselected using keyboard controls
